src/Filter/StringFilter.php

```diff
@@ -29,13 +34,14 @@ class StringFilter extends Filter
 
     public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
     {
-        if (!$data || !\is_array($data) || !\array_key_exists('value', $data) || null === $data['value']) {
+        if (!$data || !\is_array($data) || !\array_key_exists('value', $data)) {
```

Comment:
```suggestion
        if (!\isset($data['value']) && !\in_array($data['value'], [null, ''])) {
```

Reply:
@peter-gribanov want to allow the value `''`/`null`.

Reply:
`null` and empty string is available values.